vcd_cli/vapp.py

```diff
@@ -176,7 +175,7 @@ def list_vapps(ctx):
     try:
         client = ctx.obj['client']
         result = []
-        resource_type = 'adminVApp' if is_admin(ctx) else 'vApp'
+        resource_type = 'adminVApp' if is_sysadmin(ctx) else 'vApp'
```

Comment:
You mean client.is_sysadmin()? Please get rid of is_sysadmin(ctx) method as part of this change. It is no longer necessary.

Reply:
I actually thought of doing that. But I have reservation against exposing the client in pyvcloud directly to the vcd-cli. I know that we have used the client directly to list orgs, but it seems like a bad idea to me.
pyvcloud shouldn't expose how it is communicating with vCD, all that matters to the consumers of pyvcloud is that they issue a command to *pyvcloud* and they get back result _magically_.

Reply:
Makes sense.